client/ayon_core/hosts/maya/plugins/load/load_image_plane.py

```diff
@@ -145,6 +145,18 @@ def load(self, context, name, namespace, data, options=None):
                 fileName=context["representation"]["data"]["path"],
```

Comment:
This PR didn't work for me until I made that change on my side. 
```suggestion
                fileName=context["representation"]["attrib"]["path"],
```

Reply:
Interesting - that's actually wrong too! :)

```suggestion
                fileName=self.filepath_from_context(context),
```

Reply:
@MustafaJafar could you retry now with https://github.com/ynput/ayon-core/pull/246/commits/e78b8043ab9bf2e4e44d1e545e0ed64b9745c3c4